src/core/View.js

```diff
@@ -166,6 +166,8 @@ export default class View extends Component {
     this.resetCamera = this.resetCamera.bind(this);
     const bbox = vtkBoundingBox.newInstance({ bounds: [0, 0, 0, 0, 0, 0] });
     this.updateCubeBounds = () => {
+      if (this.props.showCubeAxes !== true) return;
```

Comment:
Should we do that instead?

```
if (!this.props.showCubeAxes) {
  return;
}
```

Reply:
Matter of preferences I suppose, your suggestion will cast any thruty value to false, while mine will explicitly check for a `true` boolean. I updated my PR to use your version though.

Reply:
Thanks. I think that I use the loose test everywhere else so just to be on the safe side I rather let it be anything. Thanks for the edit.